aeon_common.proto

```diff
@@ -0,0 +1,185 @@
+syntax = "proto3";
+
+package aeon;
+
+option go_package = "./pb";
+
+// Common API to Aeon - a distributed database based on Tarantool.
+service AeonCommonService {
```

Comment:
I think we should put the `Ping` request into a separate service, for example, `DiagService` or something like it (I haven't really though about a name). Maybe, split the common proto into more parts (schema, value, tuple, error)?

Reply:
i think it is worth to extract one other .proto: ddl.proto

Now we can move `create_space`/`drop_space` to the file.
Then we add `list_spaces` etc here.

Reply:
Thanks, fixed. Now we have following files:
```
aeon_crud.proto
aeon_ddl.proto
aeon_diag.proto
aeon_error.proto
aeon_schema.proto
aeon_sql.proto
aeon_value.proto
```